windows-apps-src/design/shell/tiles-and-notifications/send-local-toast.md

```diff
@@ -287,6 +287,9 @@ In the example you see below, you can retrieve the arguments string you initiall
 > [!IMPORTANT]
 > You must initialize your frame and activate your window just like your **OnLaunched** code. **OnLaunched is NOT called if the user clicks on your toast**, even if your app was closed and is launching for the first time. We often recommend combining **OnLaunched** and **OnActivated** into your own `OnLaunchedOrActivated` method since the same initialization needs to occur in both.
 
+> [!NOTE]
+> Starting WinRT 19041, MSIX packaged desktop applications should use the [OnActivated](https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.application.onactivated) for handling toast activations. For more details [ToastNotification Class](https://docs.microsoft.com/uwp/api/Windows.UI.Notifications.ToastNotification)
```

Comment:
This is incorrect too, MSIX desktop apps don't have access to OnActivated.

Ultimately, our conceptual docs don't talk about the toast.Activated += event at all, it's kind of almost strange to add a note telling them to not use it when we don't ever mention it. But if we want a note in the conceptual docs, it should say something like

"Do not use the Activated event on the ToastNotification object itself, as this event will not work if your app is closed. Instead, handle activation as we're showing on this page"

And you would add that note to both the docs here, and the desktop docs (which include MSIX). This page is purely for UWP, it says at the top that for desktop apps, they should go to the desktop docs